content/docs/demos/ingress_contour.md

```diff
@@ -33,12 +33,15 @@ osm install --set contour.enabled=true \
 
 If using `Helm`:
 ```bash
-helm install "$osm_mesh_name" osm --repo https://openservicemesh.github.io/osm
+helm install "$osm_mesh_name" osm --repo https://openservicemesh.github.io/osm \
+    --set contour.enabled=true \
+    --set contour.configInline.tls.envoy-client-certificate.name=osm-contour-envoy-client-cert \
+    --set contour.configInline.tls.envoy-client-certificate.namespace="$osm_namespace"
 ```
 
 To restrict ingress traffic on backends to authorized clients, we will set up the IngressBackend configuration such that only ingress traffic from the endpoints of the `osm-contour-envoy` service can route traffic to the service backend. To be able to discover the endpoints of `osm-contour-envoy` service, we need OSM controller to monitor the corresponding namespace.
 ```bash
-kubectl label ns "$osm_namespace" openservicemesh.io/monitored-by="$osm_mesh_name"
+osm namespace add "$osm_namespace" --mesh-name "$osm_mesh_name" --disable-sidecar-injection
```

Comment:
The short answer is to preserve the behavior of the `kubectl` command this replaces. I would need help coming up with a better reasoning than "otherwise Contour won't work." Is something like that all you would expect to see here?